mm2src/crypto/src/crypto_ctx.rs

```diff
@@ -322,16 +368,16 @@ where
     Ok((hw_device_info, hw_ctx))
 }
 
-enum HardwareWalletCtxState {
+enum FeatureInitializationState<Feature> {
```

Comment:
I guess you can name it just `InitializationState<Ctx>` 🙂

Reply:
Done